editor/src/themes/buttons/btn-groupWithSVG.scss

```
@@ -0,0 +1,144 @@
+  }
+}
+
+// Special button group with icons for image alignment
+.lime-align-btn-group {
+  .btn {
+    color: $white;
+    font-weight: $font-weight-500;
+    font-size: $font-size-sm;
+    line-height: $line-height-20;
+
+    background: $g-400;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)